tests/test_backend_api.py

```diff
@@ -0,0 +1,30 @@
+import unittest
+from grid2op.tests.aaa_test_backend_interface import AAATestBackendAPI
+
+from PandaModelsBackend import PandaModelsBackend
```

Comment:
Ok for the repo name and the class name to be camelcase, but shouldn't the import name be lowercase? https://peps.python.org/pep-0008/#package-and-module-names